libraries/apollo-api/api/apollo-api.api

```diff
@@ -633,6 +641,7 @@ public abstract interface class com/apollographql/apollo3/api/MutableExecutionOp
 	public abstract fun httpMethod (Lcom/apollographql/apollo3/api/http/HttpMethod;)Ljava/lang/Object;
 	public abstract fun sendApqExtensions (Ljava/lang/Boolean;)Ljava/lang/Object;
 	public abstract fun sendDocument (Ljava/lang/Boolean;)Ljava/lang/Object;
+	public abstract fun throwOnException (Ljava/lang/Boolean;)Ljava/lang/Object;
```

Comment:
Question: what about adding `throwOnException()` only on `ApolloClient.Builder` (and not `MutableExecutionOptions`)? Feels a bit overkill to be able to override that behaviour at the `ApolloCall`/`ApolloRequest` level and it limits the API surface changes?

Reply:
Yeah good point, this is probably overkill. Will keep it confined to `ApolloClient`  👍